src/test/end-to-end/paths/verifyRespondentDetails.js

```diff
@@ -12,5 +12,4 @@ Scenario('Verify Respondent Details', async ({I}) => {
 
     await claimantRespondentDetails(I, eventNames.CLAIMANT_RESPONDENT_DETAILS);
 
-}).tag('@biggerrefactoring');
-    //.retry(testConfig.TestRetryScenarios);
```

Comment:
Can you please tag this with RET-BAT as well...

Reply:
RET-BAT was tagged